schema/cassandra/cadence/versioned/v0.12/events_v2.cql

```diff
@@ -25,4 +25,17 @@ CREATE TABLE history_tree (
   PRIMARY KEY ((tree_id), row_type, branch_id )
 ) WITH COMPACTION = {
 'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
-};
\ No newline at end of file
+};
+
+CREATE TYPE history_branch_info (
+  branch_token                     blob,
```

Comment:
Can you put a comment on what's encoded in this branch token?

Reply:
Yes sure